src/tmdb.py

```
@@ -1637,31 +1640,68 @@ async def get_tmdb_localized_data(meta, data_type, language, append_to_response)
+        if os.path.exists(filename):
+            try:
+                async with aiofiles.open(filename, 'r', encoding='utf-8') as f:
+                    content = await f.read()
+                    try:
                         localized_data = json.loads(content)
-                else:
-                    localized_data = {}
+                    except json.JSONDecodeError as e:
+                        console.print(f"[red]Warning: JSON decode error in {filename}: {e}. Creating new file.[/red]")
+                        localized_data = {}
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)